### PR TITLE
Fat 1322

### PR DESCRIPTION
**Pull request by IhorBohdan**
https://foliotest.testrail.io/index.php?/cases/view/6650

### CODE REVIEW

File: cypress/integration/finance/funds/add-transfer-to-budget.spec.js [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+  //   Funds.selectBudgetDetails();
+  //   Funds.deleteBudgetViaActions();
+  //   InteractorsTools.checkCalloutMessage('Budget has been deleted');
+  //   Funds.checkIsBudgetDeleted();
+  //   Funds.deleteFundViaApi(firstFund.id);
+  //   Funds.deleteFundViaApi(secondFund.id);
+  //   Ledgers.deleteledgerViaApi(defaultLedger.id);
+  //   FiscalYears.deleteFiscalYearViaApi(defaultFiscalYear.id);
+  //   Users.deleteViaApi(user.userId);
+  // });
```
Comment (TetianaParanich) [inline cypress/integration/finance/funds/add-transfer-to-budget.spec.js:92]: Do you need to comment block 'after'?

File: cypress/integration/finance/funds/add-transfer-to-budget.spec.js [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+// In the future they want to change this.
+// For this reason I have already written a method for cleaning the data and I think it should be kept.
+import permissions from '../../../support/dictionary/permissions';
+import testType from '../../../support/dictionary/testTypes';
+import devTeams from '../../../support/dictionary/devTeams';
+import getRandomPostfix from '../../../support/utils/stringTools';
+import FiscalYears from '../../../support/fragments/finance/fiscalYears/fiscalYears';
+import TopMenu from '../../../support/fragments/topMenu';
+import Ledgers from '../../../support/fragments/finance/ledgers/ledgers';
+// import Users from '../../../support/fragments/users/users';
```
Comment (TetianaParanich) [inline cypress/integration/finance/funds/add-transfer-to-budget.spec.js:12]: please delete or uncomment this import

File: cypress/integration/finance/funds/add-transfer-to-budget.spec.js [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+    name: `autotest_fund2_${getRandomPostfix()}`,
+    code: getRandomPostfix(),
+    externalAccountNo: getRandomPostfix(),
+    fundStatus: 'Active',
+    description: `This is fund created by E2E test automation script_${getRandomPostfix()}`,
+  };
+  const defaultFiscalYear = { ...FiscalYears.defaultUiFiscalYear };
+  const defaultLedger = { ...Ledgers.defaultUiLedger };
+  const allocatedQuantity = '50';
+  let user;
```
Comment (TetianaParanich) [inline cypress/integration/finance/funds/add-transfer-to-budget.spec.js:29]: user is assigned a value but never used

File: cypress/integration/finance/funds/add-transfer-to-budget.spec.js [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+  //   Funds.checkIsBudgetDeleted();
+  //   Funds.deleteFundViaApi(firstFund.id);
+  //   Funds.deleteFundViaApi(secondFund.id);
+  //   Ledgers.deleteledgerViaApi(defaultLedger.id);
+  //   FiscalYears.deleteFiscalYearViaApi(defaultFiscalYear.id);
+  //   Users.deleteViaApi(user.userId);
+  // });
+
+  it('C6650 Add transfer to a budget by creating a transfer transaction (thunderjet)', { tags: [testType.criticalPath, devTeams.thunderjet] }, () => {
+    console.log(defaultFiscalYear);
```
Comment (TetianaParanich) [inline cypress/integration/finance/funds/add-transfer-to-budget.spec.js:95]: Do you need console.log?

File: cypress/support/fragments/finance/funds/funds.js [hunk 5]
```diff
@@ -146,6 +146,19 @@ export default {
     ]);
   },
 
+  transfer: (thisFund, fromFund) => {
+    cy.do([
+      Button('Actions').click(),
```
Comment (penfriend) [inline cypress/support/fragments/finance/funds/funds.js:151]: move to const variable

File: cypress/support/fragments/finance/funds/funds.js [hunk 6]
```diff
@@ -146,6 +146,19 @@ export default {
...
+      Button('Transfer').click(),
```
Comment (penfriend) [inline cypress/support/fragments/finance/funds/funds.js:152]: move to const variable

File: cypress/support/fragments/finance/funds/funds.js [hunk 7]
```diff
@@ -146,6 +146,19 @@ export default {
...
+      Button({ name: 'toFundId' }).click(),
+      SelectionOption(`${thisFund.name} (${thisFund.code})`).click(),
+      Button({ name: 'fromFundId' }).click(),
+      SelectionOption(`${fromFund.name} (${fromFund.code})`).click(),
+      TextField({ name: 'amount' }).fillIn('10'),
+      Modal({ id: 'add-transfer-modal' }).find(Button('Confirm')).click(),
```
Comment (penfriend) [inline cypress/support/fragments/finance/funds/funds.js:158]: Button('Confirm') move to const variable

File: cypress/integration/finance/funds/add-transfer-to-budget.spec.js (hunk 1 above)
Comment (IhorBohdan) [reply cypress/integration/finance/funds/add-transfer-to-budget.spec.js:92]: I described it in TO DO before the test

File: cypress/integration/finance/funds/add-transfer-to-budget.spec.js (hunk 2 above)
Comment (IhorBohdan) [reply cypress/integration/finance/funds/add-transfer-to-budget.spec.js:12]: I described it in TO DO before the test

File: cypress/support/fragments/finance/funds/funds.js (hunk 7 above)
Comment (IhorBohdan) [reply cypress/support/fragments/finance/funds/funds.js]: Done

File: cypress/support/fragments/finance/funds/funds.js (hunk 6 above)
Comment (IhorBohdan) [reply cypress/support/fragments/finance/funds/funds.js]: Done

File: cypress/support/fragments/finance/funds/funds.js (hunk 5 above)
Comment (IhorBohdan) [reply cypress/support/fragments/finance/funds/funds.js]: Done

File: cypress/integration/finance/funds/add-transfer-to-budget.spec.js (hunk 4 above)
Comment (IhorBohdan) [reply cypress/integration/finance/funds/add-transfer-to-budget.spec.js]: Deleted it)

File: cypress/integration/finance/funds/add-transfer-to-budget.spec.js (hunk 3 above)
Comment (IhorBohdan) [reply cypress/integration/finance/funds/add-transfer-to-budget.spec.js:29]: DOne

**Review by TetianaParanich — APPROVED** [review]

**Review by penfriend — APPROVED** [review]

**Pull request merged**